rtos/rtx/TARGET_CORTEX_M/cmsis_os.h

```diff
@@ -69,11 +69,15 @@
 #endif
 
 // The stack space occupied is mainly dependent on the underling C standard library
+#ifdef OS_MAINSTKSIZE
+#define WORDS_STACK_SIZE OS_MAINSTKSIZE
+#else
```

Comment:
There is 32k for application and 16k*2 on other buses. I think it would be worth allowing all 64k for application and unifying the name of regions such that tools like GCC could do a proper placement but ARM and IAR tools can take advantage of multi-region placement.